test/BitmapTests.cpp

```diff
@@ -0,0 +1,96 @@
+#include <gtest/gtest.h>
+#include <Gosu/Bitmap.hpp>
+
+namespace Gosu
+{
+    class BitmapTests : public testing::Test
+    {
+    };
+
+    static testing::AssertionResult visible_pixels_are_equal(const Bitmap& lhs, const Bitmap& rhs)
```

Comment:
I'm not really happy with the crazy indentation of these tests. Maybe I'll reduce the namespace indent from 4 to 0, after all.